tests/integration/models/rack_test.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from nav.models.manage import Sensor
+from nav.models.rack import Rack, SensorRackItem
+
+
+class TestRack:
+    def test_that_rack_configuration_can_be_saved_without_error(self, test_rack):
+        test_rack.save()
+        assert test_rack.id
+
+    def test_that_rack_configuration_is_a_dict(self, test_rack):
+        test_rack.save()
+        rack = Rack.objects.get(id=test_rack.id)
+        print(repr(Rack._configuration))
```

Comment:
Is the print statement left here on purpose?

Reply:
More or less, @johannaengland.

*pytest* will consume any printed output from your tests, and display it only when the test fails. Sometimes this becomes really useful to get more debugging information up-front from failing tests.